### ISSUE TITLE
Submit to package registry (OpenUPM?)

### ISSUE THREAD

**Issue by NoTuxNoBux** [wackoisgod/com.unity.ide.vscode#4]
At some point this can become the de facto community maintained VSCode package, in which case it would be good to have this in a registry somewhere so it doesn't need to be pulled over Git. [OpenUPM](https://openupm.com/) is probably a good fit since it's already used by many community packages and is easy to add to existing projects.

The alternative is to get access to the Unity GitHub repository, but I somehow doubt they will allow the community to publish under the official `com.unity` prefix.

**Comment by wackoisgod** [comment]
I agree that it would make sense to put it up somewhere. I have looked into OpenUPM and would likely put it there and change the name to something without the com.unity...
